api/api.proto

```diff
@@ -654,19 +644,76 @@ message StorageCluster {
   repeated StorageNode nodes = 4;
 }
 
-// Enumerate
+service OpenStorageCluster {
+	// Enumerate lists all the nodes in the cluster.
+	rpc Enumerate(ClusterEnumerateRequest)
+		returns (ClusterEnumerateResponse) {}
+
+	// Inspect the node given a UUID.
+	rpc Inspect(ClusterInspectRequest)
+    returns (ClusterInspectResponse) {}
+    
+  // Get a list of alerts from the storage cluster
+  rpc AlertEnumerate(ClusterAlertEnumerateRequest)
```

Comment:
Is this a RPC standard to name it as AlertEnumerate? IMO EnumerateAlert sounds more intuitive.

Reply:
Trying to standardize on reverse object naming style ---> direction. LIke Cluster->Alert->Enumerate.. Or AAA->BBB->CCCC for the API